### PR TITLE
Add CI test for products

### LINKED ISSUE #2132: Add secondary products to at least one CI test case
### What new functionality do you need?

There are a number of secondary products that are off by default with their own jobs that should be tested by the CI in at least one CI test.

### What are the requirements for the new functionality?

Products for the following should be turned on for at least one CI test:
- bufr
- gempak
- awips
- goes

AWIPS will need to wait until those scripts function without failure. With the addition of the missing gempak jobs, that may also need to wait until the new jobs are corrected.

### Acceptance Criteria

- [ ] At least one CI test runs postbufr
- [ ] At least one CI test runs gempak jobs (can be the same test)
- [ ] At least one CI test runs awips jobs (can be the same test)
- [ ] At least one CI test creates GOES post files (can be the same test)

### Suggest a solution (optional)

The associated `DO_*` settings in config.base will need to be updated to use the @ templates with the associated defaults added to `workflow/hosts/*`.

### LINKED ISSUE #2445: Retire AWIPS GRIB1 products
### What new functionality do you need?

AWIPS GRIB1 products (xtrn.awpgfsFFF.211.gfs_awips_fFFF_CC) will be retired in GFSv17, and thus its associated j-job JGFS_ATMOS_AWIPS_G2 and scripts can be removed.

### What are the requirements for the new functionality?

None.

### Acceptance Criteria

The xtrn.awpgfsFFF.211.gfs_awips_fFFF_CC files are no longer generated under gfs.YYYYMMDD/CC/products/atmos/wmo.

### Suggest a solution (optional)

Please let me know the process of retiring old products in the workflow. Thanks!

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
# Description
Adds a new version of the atm3DVar test that runs the full forecast length and produces most of the secondary products. For now, this test will only run on WCOSS due to gempak failures on other machines as well as computational needs. On other machines, the original version will run (the original version will not run on WCOSS). AWIPS remains off for now in this extended test due to a bug involving tocgrib2 and the convective precip fields (see #2566).

The new test runs for 4½ cycles and the full 384-hr forecast length to ensure all gempak scripts are exercised. Since the cycle throttle is 3 and the bulk of the time is in the free forecast, the cycles run mostly concurrently so it doesn't extend the total test time too much beyond that of a single 384-hr forecast.

Fixes a bug in NPOESS that was introduced when the post filenames were reverted to the previous format for the GOES products until the final filenames are determined (#2499).

Also removes the AWIPS g2 job from the rocoto mesh to complete the retirement of grib1 products.

Resolves #2132
Resolves #2445 

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
- New test run on WCOSS

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
Keeping this draft for now while manual testing runs again on WCOSS. I ran it once and it passed other than AWIPS, but I'm checking again now that I've disabled that in the test.

File: ci/cases/pr/C96_atm3DVar_extended.yaml
Comment (TerrenceMcGuinness-NOAA) [inline ci/cases/pr/C96_atm3DVar_extended.yaml:1]: Looks good.  Are we ready to run it in CI BASH on WCOSS2?

**Review by TerrenceMcGuinness-NOAA — APPROVED** [review]

**Review by GwenChen-NOAA — APPROVED** [review]
@WalterKolczynski-NOAA, your changes look good to me. I will be on travel next week, so I won't be able to validate your test run. I don't want to make you wait, so I jump ahead and approve this PR. I will check after I'm back to the office on May 13, if this PR is still open. Thanks!

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by WalterKolczynski-NOAA** [comment]
> Experiment C96_atmaerosnowDA **FAILED** on Hera in `/scratch1/NCEPDEV/global/CI/2567/RUNTESTS/C96_atmaerosnowDA_dcd36443`

@TerrenceMcGuinness-NOAA These paths are missing a `COMROOT`

**Comment by WalterKolczynski-NOAA** [comment]
`C96_atmaerosnowDA` failed the first time after the R version change due to a random rocoto error. Succeeded the second time, but it wasn't reported back correctly:

```
hfe11 C96_atmaerosnowDA_dcd36443/ > rocotostat -d C96_atmaerosnowDA_dcd36443.db -w C96_atmaerosnowDA_dcd36443.xml 
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202112201200                gdasfcst                    59642686           SUCCEEDED                   0         1         759.0
202112201200    gdasatmos_prod_f000-f003                    59642987           SUCCEEDED                   0         1         155.0
202112201200    gdasatmos_prod_f006-f009                    59643116           SUCCEEDED                   0         1         157.0
202112201200                gdasarch                    59643491           SUCCEEDED                   0         1          70.0
================================================================================================================================
202112201800                gdasprep                    59643490           SUCCEEDED                   0         1         243.0
202112201800                gdasanal                    59643701           SUCCEEDED                   0         1        1098.0
202112201800              gdassfcanl                    59644307           SUCCEEDED                   0         1         157.0
202112201800            gdasanalcalc                    59644600           SUCCEEDED                   0         1          55.0
202112201800         gdasprepsnowobs                    59643700           SUCCEEDED                   0         1          60.0
202112201800             gdassnowanl                    59643937           SUCCEEDED                   0         1          72.0
202112201800            gdasanaldiag                    59644306           SUCCEEDED                   0         1         126.0
202112201800           gdasatmanlupp                    59644694           SUCCEEDED                   0         1          32.0
202112201800          gdasatmanlprod                    59644863           SUCCEEDED                   0         1          78.0
202112201800                gdasfcst                    59644599           SUCCEEDED                   0         1         734.0
202112201800    gdasatmos_prod_f000-f003                    59644864           SUCCEEDED                   0         1         155.0
202112201800    gdasatmos_prod_f006-f009                    59645069           SUCCEEDED                   0         1         158.0
202112201800             gdasfit2obs                    59652811           SUCCEEDED                   0         1          18.0
202112201800             gdasverfozn                    59644601           SUCCEEDED                   0         1          39.0
202112201800             gdasverfrad                    59644598           SUCCEEDED                   0         1         533.0
202112201800             gdasvminmon                    59644305           SUCCEEDED                   0         1          18.0
202112201800                gdasarch                    59652864           SUCCEEDED                   0         1          79.0
202112201800             gdascleanup                    59652920           SUCCEEDED                   0         1          22.0
================================================================================================================================
202112210000                gdasprep                    59652812           SUCCEEDED                   0         1         260.0
202112210000                gdasanal                    59652918           SUCCEEDED                   0         1        1323.0
202112210000              gdassfcanl                    59653579           SUCCEEDED                   0         1         158.0
202112210000            gdasanalcalc                    59653680           SUCCEEDED                   0         1          54.0
202112210000         gdasprepsnowobs                    59652916           SUCCEEDED                   0         1          55.0
202112210000             gdassnowanl                    59653009           SUCCEEDED                   0         1          66.0
202112210000            gdasanaldiag                    59653580           SUCCEEDED                   0         1         129.0
202112210000           gdasatmanlupp                    59653771           SUCCEEDED                   0         1          39.0
202112210000          gdasatmanlprod                    59653892           SUCCEEDED                   0         1          77.0
202112210000                gdasfcst                    59653679           SUCCEEDED                   0         1         735.0
202112210000    gdasatmos_prod_f000-f003                    59653893           SUCCEEDED                   0         1         154.0
202112210000    gdasatmos_prod_f006-f009                    59654074           SUCCEEDED                   0         1         154.0
202112210000             gdasfit2obs                    59654295           SUCCEEDED                   0         1          18.0
202112210000             gdasverfozn                    59653681           SUCCEEDED                   0         1          37.0
202112210000             gdasverfrad                    59653682           SUCCEEDED                   0         1         691.0
202112210000             gdasvminmon                    59653581           SUCCEEDED                   0         1          20.0
202112210000                gdasarch                    59654375           SUCCEEDED                   0         1          86.0
202112210000             gdascleanup                    59654464           SUCCEEDED                   0         1          22.0
202112210000                 gfsprep                    59652813           SUCCEEDED                   0         1         244.0
202112210000                 gfsanal                    59652917           SUCCEEDED                   0         1         867.0
202112210000               gfssfcanl                    59653278           SUCCEEDED                   0         1         156.0
202112210000             gfsanalcalc                    59653482           SUCCEEDED                   0         1          53.0
202112210000          gfsprepsnowobs                    59652919           SUCCEEDED                   0         1          55.0
202112210000              gfssnowanl                    59653008           SUCCEEDED                   0         1          66.0
202112210000            gfsatmanlupp                    59653582           SUCCEEDED                   0         1          36.0
202112210000           gfsatmanlprod                    59653683           SUCCEEDED                   0         1         128.0
202112210000                 gfsfcst                    59653481           SUCCEEDED                   0         1        4264.0
202112210000    gfsatmos_prod_f000-f006                    59653684           SUCCEEDED                   0         1         391.0
202112210000    gfsatmos_prod_f009-f015                    59653770           SUCCEEDED                   0         1         392.0
202112210000    gfsatmos_prod_f018-f024                    59653894           SUCCEEDED                   0         1         395.0
202112210000    gfsatmos_prod_f027-f033                    59654075           SUCCEEDED                   0         1         389.0
202112210000    gfsatmos_prod_f036-f042                    59654296           SUCCEEDED                   0         1         390.0
202112210000    gfsatmos_prod_f045-f051                    59654376           SUCCEEDED                   0         1         389.0
202112210000    gfsatmos_prod_f054-f060                    59654465           SUCCEEDED                   0         1         398.0
202112210000    gfsatmos_prod_f063-f069                    59654547           SUCCEEDED                   0         1         390.0
202112210000    gfsatmos_prod_f072-f078                    59654839           SUCCEEDED                   0         1         398.0
202112210000    gfsatmos_prod_f081-f087                    59654939           SUCCEEDED                   0         1         392.0
202112210000    gfsatmos_prod_f090-f096                    59655075           SUCCEEDED                   0         1         390.0
202112210000    gfsatmos_prod_f099-f105                    59655329           SUCCEEDED                   0         1         395.0
202112210000    gfsatmos_prod_f108-f114                    59655526           SUCCEEDED                   0         1         395.0
202112210000    gfsatmos_prod_f117-f120                    59655527           SUCCEEDED                   0         1         263.0
202112210000              gfsvminmon                    59653279           SUCCEEDED                   0         1          18.0
202112210000              gfstracker                    59655784           SUCCEEDED                   0         1          69.0
202112210000              gfsgenesis                    59655785           SUCCEEDED                   0         1         549.0
202112210000                 gfsarch                    59655925           SUCCEEDED                   0         1         223.0
202112210000              gfscleanup                    59656081           SUCCEEDED                   0         1          25.0
```

**Pull request merged**